oev-litepaper.tex

```diff
@@ -89,7 +89,7 @@ \section{MEV and Oracles}
 \subsection{Frontrunning}
 
 Frontrunning and arbitrage are known as \textit{toxic flow} for derivative protocols because these transactions lead solely to losses for their LPs without providing any meaningful benefits. 
-Large data feed customers such as Synthetix have outlined the issue frontrunning poses for LP profitability in proposal~\cite{sip-37}.
+Large data feed customers such as Synthetix have outlined the issue frontrunning poses for LP profitability~\cite{sip-37}.
```

Comment:
IEEE style allows references to be treated as nouns (https://ieeeauthorcenter.ieee.org/wp-content/uploads/IEEE-Reference-Guide.pdf) but it may be the only one